client/README.md

```diff
@@ -174,8 +174,10 @@ Class | Method | HTTP request | Description
  - [CreateAksProperties](docs/CreateAksProperties.md)
  - [CreateAksPropertiesAks](docs/CreateAksPropertiesAks.md)
  - [CreateAmazonObjectStoreBucketProperties](docs/CreateAmazonObjectStoreBucketProperties.md)
+ - [CreateAmazonObjectStoreBucketPropertiesAmazon](docs/CreateAmazonObjectStoreBucketPropertiesAmazon.md)
```

Comment:
Is this name correct?

Reply:
This file is generated from OpenAPI descriptor.

Reply:
Yeah, but I wonder why there is `CreateAmazonObjectStoreBucketProperties` and `CreateAmazonObjectStoreBucketPropertiesAmazon` as well